types/kuzzle_response.go

```diff
@@ -223,7 +215,7 @@ type (
 	CredentialStrategyFields []string
 	CredentialFields         map[string]CredentialStrategyFields
 
-	Credentials map[string]interface{}
+	Credentials = json.RawMessage
```

Comment:
Why hiding `json.RawMessage` behind a new type name?
I don't know if there is any added value to that, but I'm pretty sure this makes life a lot harder to SDK users

Reply:
I don't remove this type for now because it is used elsewhere where we have to refactor also. I will remove it once we finished all the controller